e2e/deployment.go

```diff
@@ -254,3 +262,37 @@ func (yrn *yamlResourceNamespaced) Do(action kubectlAction) error {
 
 	return nil
 }
+
+type rookNFSResource struct {
+	f           *framework.Framework
+	modules     []string
+	orchBackend string
+}
+
+func (rnr *rookNFSResource) Do(action kubectlAction) error {
+	if action != kubectlCreate {
+		// we won't disabled modules
+		return nil
+	}
+
+	for _, module := range rnr.modules {
+		cmd := fmt.Sprintf("ceph mgr module enable %s", module)
+		_, _, err := execCommandInToolBoxPod(rnr.f, cmd, rookNamespace)
+		if err != nil {
+			// depending on the Ceph/Rook version, modules are
+			// enabled by default
+			e2elog.Logf("enabling module %q failed: %v", module, err)
+		}
+	}
+
+	if rnr.orchBackend != "" {
+		// this is not required for all Rook versions, allow failing
+		cmd := fmt.Sprintf("ceph orch set backend %s", rnr.orchBackend)
+		_, _, err := execCommandInToolBoxPod(rnr.f, cmd, rookNamespace)
+		if err != nil {
+			e2elog.Logf("setting orch backend %q failed: %v", rnr.orchBackend, err)
+		}
+	}
+
+	return nil
+}
```

Comment:
This should be handled as part of `scripts/rook`? it helps in setting up the test cluster with just scripts

Reply:
Rook 1.9+ should not need this, I think. Currently `scripts/rook.sh` does not execute any commands inside the toolbox to deploy/configure something.

With the changes here, testing on an ODF-4.10 deployment was possible for me. If it would be placed in `rook.sh`, I would have needed (more) manual steps.